### PR TITLE
[gpctl] Support batch-building workspace images

### PR DESCRIPTION
**Pull request by csweichel**
This PR adds support for batch-building workspace images. This is particularly handy just before a deployment to pre-build the most used workspace images.

For example, the following will build the workspace images from all workspace instances started in the last 30 days.
```
mysql -N -B -u gitpod -p -h 127.0.0.1 gitpod \
	-e 'SELECT ws.baseImageNameResolved FROM d_b_workspace_instance wsi LEFT JOIN d_b_workspace ws ON ws.id = workspaceId WHERE wsi.creationTime > (NOW() - INTERVAL 30 DAY)' | \
	sort | \
	uniq | \
	gpctl imagebuilds build-batch
```

### CODE REVIEW

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/5032#" title="Author self-approved">csweichel</a>*

*No associated issue*. Update pull-request body to add a reference to an issue, or get approval with `/approve no-issue`

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[dev/gpctl/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/dev/gpctl/OWNERS)~~ [csweichel]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by aledbf — APPROVED** [review]
LGTM

**Comment by roboquat** [comment]
LGTM label has been added.  <details>Git tree hash: f679f77766a965c026af075954811d52bca3033d</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/5032#pullrequestreview-719690350" title="Approved">aledbf</a>*, *<a href="https://github.com/gitpod-io/gitpod/pull/5032#" title="Author self-approved">csweichel</a>*

*No associated issue*. Update pull-request body to add a reference to an issue, or get approval with `/approve no-issue`

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[dev/gpctl/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/dev/gpctl/OWNERS)~~ [aledbf,csweichel]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: dev/gpctl/cmd/imagebuilds-build-many.go
Comment (aledbf) [inline dev/gpctl/cmd/imagebuilds-build-many.go:1]: Besides changing 2021, this LGTM

**Comment by roboquat** [comment]
New changes are detected. LGTM label has been removed.

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/5032#pullrequestreview-719690350" title="Approved">aledbf</a>*, *<a href="https://github.com/gitpod-io/gitpod/pull/5032#issuecomment-890575204" title="Approved">csweichel</a>*

Associated issue requirement bypassed by: *<a href="https://github.com/gitpod-io/gitpod/pull/5032#issuecomment-890575204" title="Approved">csweichel</a>*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[dev/gpctl/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/dev/gpctl/OWNERS)~~ [aledbf,csweichel]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by roboquat** [comment]
LGTM label has been added.  <details>Git tree hash: 01d678a35bfeaae2564721a4cbb6dd8adb86753c</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/5032#issuecomment-890575350" title="LGTM">aledbf</a>*, *<a href="https://github.com/gitpod-io/gitpod/pull/5032#issuecomment-890575204" title="Approved">csweichel</a>*

Associated issue requirement bypassed by: *<a href="https://github.com/gitpod-io/gitpod/pull/5032#issuecomment-890575204" title="Approved">csweichel</a>*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[dev/gpctl/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/dev/gpctl/OWNERS)~~ [aledbf,csweichel]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
